kitsune/sumo/static/sumo/less/main.less

```diff
@@ -1471,10 +1471,6 @@ input[type=button],
   .grid_12 {
     z-index: 1;
 
-    > * {
-      float: left;
-    }
```

Comment:
for whatever reason this existed, cause all links to float left. Removing it fixes the problem and announcement still look good

Reply:
thanks @jonisavage